src/main/java/com/example/rentalSystem/domain/member/entity/Member.java

```diff
@@ -0,0 +1,30 @@
+package com.example.rentalSystem.domain.member.entity;
+
+import jakarta.persistence.DiscriminatorColumn;
+import jakarta.persistence.DiscriminatorType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Inheritance;
+import jakarta.persistence.InheritanceType;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Entity
+@Getter
+@SuperBuilder
+@Inheritance(strategy = InheritanceType.JOINED)
+@DiscriminatorColumn(name = "member_type", discriminatorType = DiscriminatorType.STRING)
+@NoArgsConstructor
+public abstract class Member {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)  // or GenerationType.AUTO
+
+    private Long id;
```

Comment:
```suggestion
    @GeneratedValue(strategy = GenerationType.IDENTITY)  // or GenerationType.AUTO
    private Long id;
```
줄바꿈 정리해주시고, GenerationType을 IDENTITY로 지정해도 괜찮을 것 같아요. MySQL만 사용하니까.